### DISCUSSION TITLE
Error message 'Variables/c/InitialCondition1' does not have an associated "Action".

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#24572] [Q&A Modules: Phase field] [answered]
Hi everyone, 

When I ran my simulation, i got this error message.
**56: section 'Variables/c/InitialCondition1' does not have an associated "Action".
Did you misspell it?
68: section 'Variables/c/InitialCondition2' does not have an associated "Action".
Did you misspell it?
80: section 'Variables/c/InitialCondition3' does not have an associated "Action".
Did you misspell it?
92: section 'Variables/c/InitialCondition4' does not have an associated "Action".
Did you misspell it?**
What is the problem of this? How to correct it?

I attached my input file here,
[file.txt](https://github.com/idaholab/moose/files/11629998/file.txt)

Thank you very much,
Best,

**Comment by GiudGiud** [comment, marked as answer]
Hello

Please follow the tutorial again.
These are very basic questions, you are simply not following the syntax of a moose input. 

If should be something like:
```
[Variables]
  [c]
    [InitialCondition]
       type = ...
    []
  []
[]

Guillaume

**Reply by abc-hy** [reply]
I changed to this and it works now.
![image](https://github.com/idaholab/moose/assets/63679626/e521caff-df66-4d49-9fd7-8c4de444ed44)
![image](https://github.com/idaholab/moose/assets/63679626/7aa8fd1b-752b-4f2e-b644-ff2b80d27d15)

**Reply by GiudGiud** [reply]
this is fine too
